lexers/s/scdoc.go

```diff
@@ -0,0 +1,35 @@
+package s
+
+import (
+	"strings"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// scdoc lexer.
```

Comment:
```suggestion
// Scdoc lexer.
```

Reply:
For that one I've been commenting using what goes into `Name` field.